packages/amazon-sumerian-hosts-core/src/core/awspack/LexFeature.js

```diff
@@ -0,0 +1,287 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+import Deferred from 'core/Deferred';
```

Comment:
Is there a reason we're using deferred in this codebase instead of standard Promise?

Reply:
No particular reason for this class. I can switch to standard Promise